stacker/blueprints/asg.py

```diff
@@ -165,9 +165,18 @@ def create_load_balancer(self):
                     GetAtt(elb_name, 'DNSName')],
                 Condition="SetupELBDNS"))
 
+    def get_launch_configuration_parameters(self):
```

Comment:
I feel like we should come up with a standard for doing this. Anytime I have a slight divergence from one of the base blueprints, I have to either create a pull request to support a parameter that wasn't initially included or create my own subclass and copy paste.